devel/grab_files_from_stacks.rb

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env ruby
+require File.expand_path(File.dirname(__FILE__) + '/../config/boot')
+
+require 'csv'
+require 'druid-tools'
+
+# Copies files from the stacks into a separate folder given a list of druids
+# Pass into the full path to the input CSV below and the full path to the output location.
+# The input CSV must have a column called 'druid' containing the druid, other columns are ignored
+# August 29, 2019
```

Comment:
Could you please indicate who requested it and the context too?

Reply:
Yes, If we don't have tests, I would like to know, how long do we have to support this and who is in charge of authoring/testing changes that need to be made.  Ideally this would have a sunset date after which we can delete.

Reply:
added some comments/context

Reply:
Can you explain why this script has to circumvent the existing APIs to accomplish what it needs to?

Reply:
Okay, in that case, can we add "This work was done as a one time request and does not need to be maintained" in the documentation?

Reply:
Sure, done.  I'd also like to see us agree on a general strategy for these types of requested scripts to minimize time spent on them.  For example, a general README in this "devel" folder (assuming that is the specified place) that stipulates the scripts are generally unsupported and untested, and not suitable for regular production usage.